.github/workflows/assign-pull-request.yml

```diff
@@ -1,8 +1,8 @@
 name: Assign pull request
 
 on:
-  pull_request:
-    types: [opened]
+  pull_request_target:
+    types: [opened, ready_for_review]
```

Comment:
We want this to now run not only when a PR is opened, but run it again when PR marked ready for review, since by default it won't assign a reviewer to a draft PR